src/emcpy/calcs/wind.py

```diff
@@ -13,42 +13,39 @@ def uv_to_spddir(u, v, direction=False):
     then return zero.
     Parameters
     ----------
-    u, v: array_like
+        u, v: array_like
         u (easterly) and v (northerly) wind component.
-    direction: boolean (optional)
+        direction: boolean (optional)
     Returns
     -------
-    Wind speed (and direction if direction=True)
+        Wind speed (and direction if direction=True)
     """
-    u = np.array(u)
-    v = np.array(v)
+    u = _np.array(u)
+    v = _np.array(v)
 
-    wdir = (270 - np.rad2deg(np.arctan2(v, u))) % 360
-    wspd = np.sqrt(u * u + v * v)
+    wdir = (270 - _np.rad2deg(_np.arctan2(v, u))) % 360
```

Comment:
I understand that _ is convention for defining private functions, but in terms of a module using it shouldn't it not matter which is used? I've seen both (matplotlib and netCDF4 use import numpy as np for example)

Reply:
done.